src/components/organisms/modals/UserAndRoleManagementModal/UserAndRoleManagementModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, ReactElement } from 'react'
+import { AppearanceTypes, SizeTypes } from 'components/molecules/Button/Button'
+import ModalBase, {
+  ButtonPositionTypes,
+  TitleFontTypes,
+} from 'components/organisms/ModalBase/ModalBase'
+
+export interface UserAndRoleManagementModalProps {
+  title?: string
+  cancelButtonContent?: string
+  proceedButtonContent?: string
+  modalContent?: string | ReactElement
+  isModalOpen?: boolean
+  closeModal: () => void
```

Comment:
Why this closeModal prop is here, if you don't send any func with it? Are you planing to use the closeModal func?